apps/api/src/components/Authentication/OTPService/index.ts

```diff
@@ -0,0 +1,40 @@
+import authclientStytch from "../../../lib/stytch.js";
+import normalizePhoneNum from "../../../helpers/normalizePhoneNum.js";
+
+type phoneCodeReturn<Type> = {
```

Comment:
As a general rule, types begin with a capital letter. Also naming the generic argument Type may not be the best because lowercase type is a reserved word. The usual name for a generic is just T
```suggestion
type PhoneCodeReturn<T> = {
```

Reply:
noted and fixed